.NET/Microsoft.Recognizers.Text.DataDrivenTests/NetCoreTestDataSourceAttribute.cs

```diff
@@ -30,15 +30,6 @@ public IEnumerable<object[]> GetData(MethodInfo methodInfo)
             var rawData = File.ReadAllText(specsFile);
             var specs = JsonConvert.DeserializeObject<IList<TestModel>>(rawData);
 
-            // Adding a pseudo test case to avoid test failure causing by returning empty sequence.
```

Comment:
Please don't undo other people's changes unless they are part of the PR content.